air-script/tests/evaluators/evaluators.air

```diff
@@ -16,5 +18,5 @@ boundary_constraints:
     enf b.first = 0
 
 integrity_constraints:
-    enf is_unchanged([b])
+    enf are_unchanged([b, c[1], d[2]])
```

Comment:
Actually we don't yet support variables as indices yet in the parser. Maybe we should prioritize this issue since a lot of our constraints depend on this. Also, we haven't merged constraint comprehension for evaluators yet so I've added test for:
```
ev are_all_binary([c[3]]):
    enf c^2 = c for c in c
```

Reply:
Ah ok good point, thanks. That works! That's not the part I cared about for the test anyway